packages/block-editor/src/components/inserter/menu.js

```diff
@@ -101,12 +101,14 @@ function InserterMenu(
 			window.requestAnimationFrame( () => {
 				if (
 					! shouldFocusBlock &&
-					! blockTypesTabRef?.current.contains(
+					! blockTypesTabRef?.current?.contains(
```

Comment:
I think `blockTypesTabRef` is always defined

```suggestion
					! blockTypesTabRef.current?.contains(
```

Reply:
Indeed 👍 cleaned up in 112547b11005ba1eaabb64d54bfea4379f8c3e33